src/elements/content-sidebar/versions/__tests__/VersionsSidebarContainer.test.js

```diff
@@ -263,6 +265,17 @@ describe('elements/content-sidebar/versions/VersionsSidebarContainer', () => {
 
             expect(wrapper.state('isWatermarked')).toBe(true);
         });
+
+        test('should set state with isArchiveFile if file is archive file', () => {
```

Comment:
As above.

Reply:
I feel this file is too complicated to switch from enzyme to RTL in scope of this PR. I have RTL implementation for new tests, but I'm working on putting it alongside enzyme in this file. I'm having trouble, because some module mocks needed for RTL are breaking enzyme tests and vice versa

Reply:
Moved this test case to new file using RTL